src/js/components/Button/doc.js

```diff
@@ -200,6 +200,15 @@ export const themeDoc = {
     type: 'number',
     defaultValue: 0.3,
   },
+  'button.disabled.border.color': {
+    description: 'The border color when the button is disabled.',
+    type: 'string | { dark: string, light: string }',
+  },
+  'button.disabled.color': {
+    description: `Fill color for primary, label color for plain, border color 
```

Comment:
Since `border` may not be relevant here, let's refine the description. It should also address the use case of what the `color` means on the default button.